services/nrfradio.md

```diff
@@ -0,0 +1,69 @@
+# bit:radio
+
+    identifier: 0x1ac986cf
+    camel: bitRadio
+
+## Registers
+
+    rw enabled: bool @ intensity
+
+Turns on/off the radio antenna.
+
+    rw group: u8 @ 0x80
+
+Group used to filter packets
+
+    rw transmission_power = 6: u8 { min_value=1, max_value=7 } @ 0x81
+
+Antenna power to increase or decrease range.
+
+## Commands
+
+    command send_string @ 0x80 {
+        message: string { max_length: 18 }
+    }
+
+Sends a string payload as a radio message
+
+    command send_number @ 0x80 {
+        value: f64
```

Comment:
@mmoskal do we support doubles?

Reply:
I see no problem with this, especially that the JD service only need to pass on the bits, no need to do any double computations.